src/v2/Apps/Conversation/Components/ConversationCTA.tsx

```diff
@@ -67,6 +121,8 @@ export const ConversationCTAFragmentContainer = createFragmentContainer(
             ... on Artwork {
               __typename
               isOfferableFromInquiry
+              is_acquireable: isAcquireable
```

Comment:
Is there a reason for this to be aliased? 🤔

Reply:
Good question. We are using it like this in many components related to the artwork page and I was just following that. If this is not necessary I'll remove it!